src/db/mod.rs

```diff
@@ -29,7 +29,7 @@ pub struct RegistryTx<'c> {
     tx: Transaction<'c, Postgres>,
 }
 
-impl<'c> RegistryTx<'c> {
+impl RegistryTx<'_> {
```

Comment:
looks like clippy added a new lint that prevents the builds. This is what we get for not running beta and nightly on cron)

Reply:
imho not critical)